app/views/organizations/_org_info.html.erb

```diff
@@ -20,5 +20,12 @@
       <td>Twitter:</td>
       <td><%= link_to @organization.twitter,"https://twitter.com/#{@organization.twitter}" %></td>
     </tr>
+    <tr>
+      <td>Manager:</td>
+      <td>
+        <% if @organization.user %>
+        <%= @organization.user.first_name %></td>
```

Comment:
Please do two things here:
- indent
- implement a method called `user_first_name` in the `Organization` model to comply with the law of demeter